spawn/parsers/generators.py

```diff
@@ -62,7 +62,8 @@ def load_generators_from_module(module):
         :rtype: :class:`GeneratorsParser`
         """
         is_generator = lambda c: inspect.isclass(c) and issubclass(c, generator_methods.Generator)
-        return {name: class_type for name, class_type in inspect.getmembers(module, is_generator)}
+        return dict(inspect.getmembers(module, is_generator))
```

Comment:
I think I disabled this pylint warning in one of my branches - I don't really agree with it